sound/soc/codecs/rt711.h

```diff
@@ -8,6 +8,8 @@
 #ifndef __RT711_H__
 #define __RT711_H__
 
+#include <linux/soundwire/sdw.h>
+
```

Comment:
you can instead add this in the machine driver. 
@lyakh and other maintainers like it better if we include the headers when we actually need them, instead of in other include file.